### DISCUSSION TITLE
Fastest possible simulation

### DISCUSSION THREAD

**Discussion by Rixxuss** [InteractiveComputerGraphics/SPlisHSPlasH#156] [Q&A]
Hi,
which settings would you take for the fastest possible simulation?
So which solver, z-sort, boundaryHandling etc.
Kind Regards

**Comment by janbender** [comment]
Solver: DFSPH (fastest solver & optimized using AVX)
Visco: XSPH
Boundary: Volume Maps (slightly faster than Akinci)
z-sort: enabled (for many particles this leads to more cache hits)
